### PR TITLE
portuguese equivalent for XDG directories

### PR DESCRIPTION
**Pull request by LaBatata101**

### CODE REVIEW

File: devicons.py [hunk 1]
```diff
@@ -147,6 +147,16 @@
     'Images'                           : '',
     'Publique'                         : '',
     'Vidéos'                           : '',
+# Portuguese
+    'Área de trabalho'                 : '',
+    'Documentos'                       : '',
+    'Downloads'                        : '',
+    'Dropbox'                          : '',
```
Comment (alexanderjeurissen) [inline devicons.py]: This line is not adding much as the `Dropbox` key is already defined at line 136 and doesn't differ from the english definition.

File: devicons.py [hunk 2]
```diff
@@ -147,6 +147,16 @@
     'Images'                           : '',
     'Publique'                         : '',
     'Vidéos'                           : '',
+# Portuguese
+    'Área de trabalho'                 : '',
+    'Documentos'                       : '',
+    'Downloads'                        : '',
```
Comment (alexanderjeurissen) [inline devicons.py]: This line doesn't differ from the english definition and thus shouldn't be redefined.

**Comment by LaBatata101** [comment]
Fixed the redundant entries

**Comment by alexanderjeurissen** [comment]
Thanks again for your contribution.

**Pull request merged**